server.go

```diff
@@ -109,8 +110,19 @@ func (s *server) Allow(namespace, name string, tokensRequested int64, maxWaitMil
 	return w, nil
 }
 
-func (s *server) ServeAdminConsole(mux *http.ServeMux, assetsDir string, p config.ConfigPersister) {
-	admin.ServeAdminConsole(s, mux, assetsDir)
+func (s *server) ServeAdminConsole(listenerString string, assetsDir string, p config.ConfigPersister) {
+	development := os.Getenv("ENVIRONMENT") == "development"
+	msg := "Starting admin console on %s"
+
+	if development {
+		msg += " (in development mode)"
+	}
+
+	logging.Printf(msg, listenerString)
+
+	mux := http.NewServeMux()
```

Comment:
You will need to have `*http.ServeMux` passed in as a parameter, since in some environments an HTTP mux may already be running.